packages/devtools_test/lib/src/mocks/fake_isolate_manager.dart

```diff
@@ -14,6 +14,12 @@ import 'package:vm_service/vm_service.dart';
 import 'generated.mocks.dart';
 
 base class FakeIsolateManager extends Fake with TestIsolateManager {
+  FakeIsolateManager({
+    this.rootLibrary,
+  });
+
+  String? rootLibrary;
```

Comment:
pass this all the way through to line 60.  We can add a default value for `rootLibrary` 

```
  FakeIsolateManager({
    this.rootLibrary = 'package:fake_app/main.dart',
  });

  final String rootLibrary;
  ```

and then below in the `isolateState` method: 
`final rootLib = LibraryRef(id: '0', uri: rootLibrary);`

Then you may not have to override `rootLibraryForMainIsolate` here at all.

Reply:
Passing it now to the isolateState method, but switching over to using the mock service connection manager instead of the fake service connection manager required mocking a lot more of the functionality